src/main/resources/kibana-rest.yml

```
@@ -208,7 +220,9 @@ kibana_user:
   versions:
     ">= 7.6.0": "/internal/security/me"
     ">= 5.0.0 < 7.6.0": "/api/security/v1/me"
+    # ">= 9.0.0": "uri [/internal/security/me] with method [get] exists but is not available with the current configuration"
 
 kibana_stack_monitoring_health:
   versions:
     ">= 8.3.1": "/api/monitoring/v1/_health"
+    # ">= 9.0.0": "uri [/api/monitoring/v1/_health] with method [get] exists but is not available with the current configuration"
```

Does this mean we should add `< 9.0.0` to the above line to disable it?

```suggestion
    ">= 8.3.1 < 9.0.0": "/api/monitoring/v1/_health"
    # ">= 9.0.0": "uri [/api/monitoring/v1/_health] with method [get] exists but is not available with the current configuration"
```